docs/changelog/3_x.rst

```diff
@@ -0,0 +1,517 @@
+invoked them. Mutation rewrites are built to do just that!
+
+.. code-block:: sdl
+
+    type Post {
+      required title: str;
+      required body: str;
+      modified: datetime {
+        rewrite insert, update using (datetime_of_statement())
+      }
+    }
+
+This shows one reason mutation rewrites is one of our most wanted features:
+modified timestamps! When a user inserts or updates a ``Post``, the rewrite
+will set the value of the ``modified`` property to that value of
+``datetime_of_statement()``. There are tons of other uses too. Give them a try!
+
+
+Splats
+------
+
+This is one of the most fun features in 3.0, both to say _and_ to use! With
+splats, you can easily select all properties in your queries without typing all
+of them out.
+
+.. code-block:: edgeql-repl
+
+    db> select Movie {*};
+    {
+      default::Movie {
+        id: 6fe5c3ec-b776-11ed-8bef-3b2fba99fe8a,
+        release_year: 2021,
+        title: 'Spiderman: No Way Home',
+      },
+      default::Movie {
+        id: 76998656-b776-11ed-8bef-237907a987fa,
+        release_year: 2008,
+        title: 'Iron Man'
+      },
+    }
+
+You can even use a double-splat to select a single level of nested objects.
+
+.. code-block:: edgeql-repl
+
+    db> select Movie {**};
+    {
+      default::Movie {
+        id: 6fe5c3ec-b776-11ed-8bef-3b2fba99fe8a,
+        release_year: 2021,
+        title: 'Spiderman: No Way Home',
+        characters: {
+          default::Hero {
+            id: 01d9cc22-b776-11ed-8bef-73f84c7e91e7,
+            name: 'Spiderman'
+          },
+          default::Villain {
+            id: efa2c4bc-b777-11ed-99eb-43f835d79384,
+            name: 'Electro'
+          },
+          default::Villain {
+            id: f2c99a96-b775-11ed-8f7e-0b4a4a8e433e,
+            name: 'Green Goblin'
+          },
+          default::Villain {
+            id: f8ca354a-b775-11ed-8bef-273145019e1d,
+            name: 'Doc Ock'
+          },
+        },
+      },
+      default::Movie {
+        id: 76998656-b776-11ed-8bef-237907a987fa,
+        release_year: 2008,
+        title: 'Iron Man',
+        characters: {
+          default::Hero {
+            id: 48edcf8c-b776-11ed-8bef-c7d61b6780d2,
+            name: 'Iron Man'
+          },
+          default::Villain {
+            id: 335f4104-b777-11ed-81eb-ab4de34e9c36,
+            name: 'Obadiah Stane'
+          },
+        },
+      },
+    }
+
+It's a super-handy way to quickly explore your data.
+
+
+Nested modules
+--------------
+
+"Yo, dawg. We heard you like modules…" (Sorry. We've been working so hard on
+3.0, `we got way way behind on our memes
+<https://knowyourmeme.com/memes/xzibit-yo-dawg>`_.)
+
+You can now put a module inside another module to let you organize your schema
+in any way that makes sense to you.
+
+.. code-block:: sdl
+
+    module MommaModule {
+      module BabyModule {
+        <schema-declarations>
+      }
+    }
+
```

Comment:
Mention that this will allow us to expand the list of standard modules in the future in a backwards compatible way.

Reply:
Also need to add an example of how this looks like in EdgeQL